utils/cat_helper.py

```diff
@@ -3,6 +3,10 @@
 from lxml.etree import XMLSyntaxError
 import re
 
+# Define names matching Institution ERType values exactly as expected by CAT API
+# https://github.com/GEANT/CAT/blob/master/core/IdP.php#L55-L57
+ERTYPE_CAT_API_NAMES = { 1: 'IdP', 2: 'SP', 3: 'IdPSP' }
```

Comment:
We already have `edumanage.models.get_ertype_string` to translate a numeric institution type to string, which was required for eduroam DB (EDB) v2.

There is a minor inconsistency however between IdP+SP in EDB vs. IdPSP in CAT. But I think this does not warrant having a completely separate mapping for the same thing.

I will modify your patch to use the existing facility.